README.md

```diff
@@ -4,7 +4,7 @@
 [![image](https://img.shields.io/pypi/v/uv.svg)](https://pypi.python.org/pypi/uv)
 [![image](https://img.shields.io/pypi/l/uv.svg)](https://pypi.python.org/pypi/uv)
 [![image](https://img.shields.io/pypi/pyversions/uv.svg)](https://pypi.python.org/pypi/uv)
-[![Actions status](https://github.com/astral-sh/uv/workflows/CI/badge.svg)](https://github.com/astral-sh/uv/actions)
+![Actions status](https://github.com/astral-sh/uv/actions/workflows/ci/badge.svg)
```

Comment:
Currently, badge broken to show. I think you missed `.yml`:
```suggestion
![Actions status](https://github.com/astral-sh/uv/actions/workflows/ci.yml/badge.svg)
```

Reply:
Checked the wrong preview, thanks